velox/vector/arrow/Bridge.cpp

```diff
@@ -1713,6 +1866,18 @@ VectorPtr importFromArrowImpl(
 
   // String data types (VARCHAR and VARBINARY).
   if (type->isVarchar() || type->isVarbinary()) {
+    // Import StringView from Utf8View/BinaryView (Zero-copy)
```

Comment:
Can we move this check and logic into `createStringFlatVector()` to encapsulate the string conversion logic there? From there we can call `createStringFlatVectorFromUtf8View()`

Reply:
I think this is a better place to check the schema format. Moving it to createStringFlatVector() would mean adding schema format to its signature.